tests/mocha/server/CollectionsTest.js

```diff
@@ -0,0 +1,29 @@
+MochaWeb.testOnly(function() {
+  describe('Collections', function() {
+
+    Gizmos = new Mongo.Collection("gizmos");
```

Comment:
Just curious, does this work if you make Gizmos non global (give it a var keyword)?